src/main/java/org/jitsi/jicofo/VersionBroadcaster.java

```diff
@@ -171,18 +172,16 @@ public void handleEvent(Event event)
                     + "(" + jicofoVersion.toString() + ","
                     + System.getProperty("os.name") + ")");
 
-        // Videobridge
-        // It is not be reported for FOCUS_JOINED_ROOM_TOPIC
-        Jid bridgeJid
-                = (Jid)event.getProperty(EventFactory.BRIDGE_JID_KEY);
-        Version jvbVersion
-            = bridgeJid == null
-                ? null : meetServices.getBridgeVersion(bridgeJid);
-        if (jvbVersion != null)
+        String jvbVersions = conference.getBridges().stream()
```

Comment:
Fun fact: `CONFERENCE_ROOM_TOPIC` is used when a new colibri conference is allocated. I had to follow 4 references (including an exaustive search on uses of a constant) to find this out, and it is not what I expected it to be based on the name. Also, (a comment I left 2 years ago)[https://github.com/jitsi/jicofo/blob/master/src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java#L2423] suggests that at the time I failed to understand why we fire the event. Luckily I failed to convince myself that we don't need it either, so I left it in. Using `EventAdmin` leads to hard to maintain code. 

I'm just ranting, I don't expect any change. Though I would like to refactor this eventually.